src/screens/ProfileScreen/index.js

```diff
@@ -197,6 +198,8 @@ const ProfileScreen = ({route}) => {
     reloadFetchAnonymousPost,
     getProfileCache
   } = useProfileScreenHook();
+
+  const {fetchNextFeeds} = useFeedPreloadHook(mainFeeds?.length, () => getMyFeeds(postOffset));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useFeedPreloadHook` is being used here to fetch the next set of feeds when necessary. It's important to ensure that the `getMyFeeds` function passed as the callback to `useFeedPreloadHook` handles errors properly and doesn't cause any side effects if called multiple times.



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
closed